api/AbortPaymentEvent.json

```diff
@@ -0,0 +1,191 @@
+{
+  "api": {
+    "PaymentRequestEvent": {
```

Comment:
AbortPaymentEvent?

Reply:
Also, where is this interface specc'ed?

Reply:
Found it in w3c/payment-handler#207.

Reply:
Our engineer says it's still under discussion. Want me to pull it out?